### ISSUE TITLE
False positive "H1 heading cannot exceed 2 elements (react-a11y-accessible-headings)" for switch-case

### ISSUE THREAD

**Issue by KrzysztofKarol** [joaovieira/tslint-react-a11y#5]
Hi

### Thank You part
Great package! Thank You!

### Problem part
```jsx
declare var letter: 'a' | 'b' | 'c'

const getHeader = () => {
  switch (letter) {
    case 'a':
      return <h1>a</h1>
    case 'b':
      return <h1>b</h1>
    case 'c':
      return <h1>c</h1>
  }
}
```

gives `H1 heading cannot exceed 2 elements (react-a11y-accessible-headings)` but it shouldn't
